pkgs/by-name/ge/gex/package.nix

```diff
@@ -29,9 +32,6 @@ rustPlatform.buildRustPackage rec {
       darwin.apple_sdk.frameworks.Security
     ];
 
-  # force the libgit2-sys crate to use the system libgit2 library
-  LIBGIT2_NO_VENDOR = 1;
```

Comment:
can you point to upstream `git2-rs` why we don't need this? my understanding is that this env needs to be set otherwise libgit2 will be compiled from vendored sources in the libgit2-sys crate

Reply:
It seems to support our libgit2 in HEAD — could we just backport the patch that updated git2?